app/views/download/help_improve.html.haml

```diff
@@ -16,41 +16,30 @@
 = content_for :javascript do
   :javascript
     (function(document, window, undefined) {
-      var forms = document.querySelectorAll('.js--ga-submit')
-
-      var getValue = function(element) {
-          if (element === null) {
-            return 'Blank'
-          }
-          else if (element.value) {
-            return element.value
-          }
-
-          return false;
+      var isIE = function() {
+        var myNav = navigator.userAgent.toLowerCase()
+        return (myNav.indexOf('msie') != -1) ? parseInt(myNav.split('msie')[1]) : false
       }
 
-      for (var i = 0; i < forms.length; i += 1) {
-        forms[i].addEventListener('submit', function(event) {
-          var form = event.target
+      if (isIE() && isIE() >= 9 || isIE() === false ) {
```

Comment:
What's the functionality that's not available in old IE? And can we change this to check whether that's available, instead of parsing the user agent?